label/label.go

```diff
@@ -94,6 +103,17 @@ func (m *GormLabelRepository) List(ctx context.Context, spaceID uuid.UUID) ([]La
 	return objs, nil
 }
 
+// CheckExists returns nil if the given ID exists otherwise returns an error
+func (m *GormLabelRepository) CheckExists(ctx context.Context, id string) error {
```

Comment:
Do we need to make this an exported name?

Reply:
I don't see a need for this function with an indirection and no other logic.

Reply:
yes, done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/b7e1fb988030995eaf81ec06d16098df02fbd849